modules/size/size.resolver.js

```diff
@@ -4,15 +4,15 @@ const RuleError = require('../../errors/rule.error');
 const sizeQuery = {
   getAllSizes: async (_, { limit, skip, filter }) => {
     try {
-      return await sizeService.getAllSizes(limit, skip, filter);
```

Comment:
Firstly I deleted await there and eslint didn't show me that this is redundant. All tests passed. 
But after some doubts googled this:
```
// In this example the `await` is necessary to be able to catch errors thrown from `bar()`
async function foo() {
    try {
        return await bar();
    } catch (error) {}
}
```

Reply:
GO TO ZOOM